Phonics/View Controllers/LettersViewController.swift

```diff
@@ -36,12 +43,21 @@ class LettersViewController: UIViewController, UICollectionViewDataSource, UICol
     //MARK: - Collection View Data Source
     
     func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
-        return PHLetters.count
+        if difficulty == .easyDifficulty {
+            return PHLetters.count
+        }
+        
+        return phonics.count
```

Comment:
Just access it through the singleton instead. `PHContent.allPhonicsSorted.count`